tests/integration/start_hello.bats

```diff
@@ -37,6 +37,36 @@ function teardown() {
 	[[ "${output}" == *"Hello"* ]]
 }
 
+# https://github.com/opencontainers/runc/issues/3715, https://go.dev/issue/58552
+@test "runc run as user with no exec bit but CAP_DAC_OVERRIDE set" {
+	requires root # Can't chown/chmod otherwise.
+	case "$(__runc --version | grep '^go:')" in
+	"go: go1.20" | "go: go1.20.1")
+		skip "go 1.20[.1] is broken; see https://go.dev/issue/58552"
```

Comment:
Do we still need to take go 1.20 - 1.20.1 into account here? Or could we just assume; our CI will be on 1.20.2 (and if others don't, then it's their problem?)

Reply:
It took me some time to write this :) so I am hesitant as to removing this.

But you are right -- it is better to assume that Go 1.20 >= 1.20.2 is used, and we should fail otherwise.

I am replacing this one with a mere comment.